README.md

```diff
@@ -39,3 +39,13 @@ The project also draws inspiration from:
 
 The app is running in a hobby instance of vercel:
 [Production instance](https://voice-trainer-kg8920zu9-pisti2010.vercel.app/)
+
+
+## Fernando-pr - Branch push troubleshooting
+Added line 45 to push to Fernando-pr branch. 
+
+While attempting to push the README.md edits to the appropriate branch, I ran into some issues, but they were fixed by:
```

Comment:
This is a good way to write what you did, as in reporting your progress, but it's not written in proper format for a Readme.
Consider that readmes are guides for future devs, or even yourself once you forget, on how to do certain things.
Attempt to rewrite it more as instructions on how to solve the problem that you encountered, and instead feel free to take this text and copy paste it onto the PR discussion or description.

Example, you can change the readme to be worded like:
- Avoid using characters like ' and " on branch names
- To create a new branch from your current branch, do X Y Z